src/Identity/Specification.Tests/src/UserManagerSpecificationTests.cs

```diff
@@ -644,9 +644,9 @@ public async Task CanCreateUserAddRemoveLogin()
         var logins = await manager.GetLoginsAsync(user);
         Assert.NotNull(logins);
         Assert.Single(logins);
-        Assert.Equal(login.LoginProvider, logins.Last().LoginProvider);
-        Assert.Equal(login.ProviderKey, logins.Last().ProviderKey);
-        Assert.Equal(login.ProviderDisplayName, logins.Last().ProviderDisplayName);
+        Assert.Equal(login.LoginProvider, logins[logins.Count - 1].LoginProvider);
+        Assert.Equal(login.ProviderKey, logins[logins.Count - 1].ProviderKey);
+        Assert.Equal(login.ProviderDisplayName, logins[logins.Count - 1].ProviderDisplayName);
```

Comment:
You should be able to do `logins[^1]`, notably.

Reply:
I think this was the code fix that did the change. I used apply fix to all occurrences in the document